web/web/templates/security/login_user.html

```diff
@@ -1,149 +1,92 @@
+      background-color: #f5f5f5;
+    }
+
+    .form-signin {
+      width: 100%;
+      max-width: 330px;
+      padding: 15px;
+      margin: auto;
+    }
+
+    .form-signin .checkbox {
+      font-weight: 400;
+    }
+
+    .form-signin .form-floating:focus-within {
+      z-index: 2;
+    }
+
+    .form-signin input[type="email"] {
+      margin-bottom: -1px;
+      border-bottom-right-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+
+    .form-signin input[type="password"] {
+      margin-bottom: 10px;
+      border-top-left-radius: 0;
+      border-top-right-radius: 0;
+    }
+  </style>
+</head>
+
+<body class="text-center">
+  <main class="form-signin">
+    <form action="{{ url_for_security('login') }}" method="POST" name="login_user_form">
       {{ login_user_form.hidden_tag() }}
 
-      <div class="text-center mb-4">
-        <!--<img class="mb-4" src="" alt="" width="72" height="72">-->
+      <div class="mb-4">
         <h1>DTANM</h1>
         {% if login_user_form.email.errors %}
-          {% for error in login_user_form.email.errors %}
-            <div class="alert alert-danger">{{ error }}. If you've forgotten your login information, please contact your administrator.</div>
-          {% endfor %}
+        {% for error in login_user_form.email.errors %}
+        <div class="alert alert-danger">{{ error }}. If you've forgotten your login information, please contact your
+          administrator.</div>
+        {% endfor %}
         {% endif %}
         {% if login_user_form.password.errors %}
-          {% for error in login_user_form.password.errors %}
-            <div class="alert alert-danger">{{ error }}. If you've forgotten your login information, please contact your administrator.</div>
-          {% endfor %}
+        {% for error in login_user_form.password.errors %}
+        <div class="alert alert-danger">{{ error }}. If you've forgotten your login information, please contact your
+          administrator.</div>
+        {% endfor %}
         {% endif %}
       </div>
-      <div class="form-label-group">
-        <input type="email" name="email" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
+      <div class="form-floating">
+        <input type="email" name="email" class="form-control" id="inputEmail" placeholder="name@example.com">
         <label for="inputEmail">Email address</label>
       </div>
-
-      <div class="form-label-group">
-        <input type="password" name="password" id="inputPassword" class="form-control" placeholder="Password" required>
+      <div class="form-floating">
+        <input type="password" name="password" class="form-control" id="inputPassword" placeholder="Password">
         <label for="inputPassword">Password</label>
       </div>
+
       <div class="checkbox mb-3">
         <label>
-          <input type="checkbox" value="y" name="remember"> Remember me
+          <input type="checkbox" value="remember-me"> Remember me
```

Comment:
Unintended value change might break functionality.

Reply:
There doesn't seem to be very good documentation on this, but judging by the [source](https://github.com/mattupstate/flask-security/blob/674b18103fa8734aca71bbd084ea01e3709817ef/flask_security/forms.py#L212) this field should be called "remember_me" and the value doesn't matter (BooleanField is true for any value).